Sources/Core/Renderer/RenderCoordinator.swift

```diff
@@ -3,6 +3,55 @@ import FirebladeMath
 import MetalKit
 import DeltaCore
 
+public struct RendererError: LocalizedError {
```

Comment:
I think it'd make sense to just make `RendererError` itself an `enum` since it just has a single enum property (i.e. move the cases of `ErrorKind` onto `RendererError` itself).

Reply:
originally I wanted it to be an enum, but prior to gabriel's commit there was an error property. I assume it's okay to make it into an enum now

Reply:
latest commit makes it just an enum, thanks for pointing it out